src/cli/README.md

```diff
@@ -1863,6 +1863,68 @@ This command is only available when device enables NAT64 translator.
 Done
 ```
 
+### nat64 disable
+
+Disable NAT64 functions, including the translator and the prefix publishing.
+
+This command will reset the mapping table in the translator (if NAT64 translator is enabled in the build).
+
+This command is only available when device enables NAT64 translator or NAT64 prefix manager.
+
+```bash
+> nat64 disable
+Done
+```
+
+### nat64 enable
+
+Enable NAT64 functions, including the translator and the prefix publishing.
+
+The border routing manager should be enabled (if it is enabled in the build), and the NAT64 translator should be configured with a CIDR (if it is enabled in the build).
```

Comment:
The docs are not clear whether the user needs to enable the border routing manager and/or configure the CIDR before calling `nat64 enable` or if calling `nat64 enable` will also handle these configurations. Can you help clarify?

Reply:
Just a quick note: Ideally, whatever is in the README should also be in the @cli comments, with the idea that what's on the OT website will be the one source of truth. Could this be added to the API description, or added as a @par tag in the CLI block?